jupyter-extension/src/plugins/mount/components/Pipeline/hooks/usePipeline.tsx

```diff
@@ -13,17 +14,40 @@ export type usePipelineResponse = {
   requirements: string;
   setRequirements: (input: string) => void;
   callCreatePipeline: () => Promise<void>;
+  callSavePipeline: () => void;
   errorMessage: string;
 };
 
-export const usePipeline = (): usePipelineResponse => {
+export const usePipeline = (
+  metadata: SameMetadata | undefined,
+  saveNotebookMetaData: (metadata: any) => void,
+): usePipelineResponse => {
   const [loading, setLoading] = useState(false);
   const [pipelineName, setPipelineName] = useState('');
   const [imageName, setImageName] = useState('');
   const [inputSpec, setInputSpec] = useState('');
   const [requirements, setRequirements] = useState('');
   const [errorMessage, setErrorMessage] = useState('');
 
+  useEffect(() => {
+    console.log('use effect: metadata: ', metadata);
+    setImageName(
+      metadata?.environments.default.image_tag
```

Comment:
(this is me learning javascript) I think you can replace these conditionals with the `??` operator? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing

Reply:
lol `nullish` I <3 JS